Skills/Perk/PregnancyPainTolerance.gd

```diff
@@ -0,0 +1,25 @@
+extends PerkBase
+
+func _init():
+	id = Perk.PregnancyPainTolerance
+	skillGroup = Skill.Pregnancy
+
+func getVisibleName():
+	return "Pain tolerance"
+
+func getVisibleDescription():
+	return "Life in prison is tough especially if you are constantly being pregnant, all the challenges that you have overcome made you much sturdier. \n" \
+			+ "Resistance to physical damage +15% \n" \
+			+ "Maximum stamina +30" #maybe add some percentage total, like +10% total stamina
```

Comment:
Yeah I dunno. All the perks I make provide bonuses only under certain conditions. Nothing should be free

Reply:
> Yeah I dunno. All the perks I make provide bonuses only under certain conditions. Nothing should be free

Make giving birth to like 50-100 children as an additional requirement?

Reply:
Actually there are quite a few "free" bonuses in combat and some other perk trees. Lowered physical resistance to 10% for now. It's not really that much for a T3 perk. BDSM perk is giving 15% but is T1 although with a condition. So I think this is balanced. But adding some req like 50-100 children born is possible of course.